replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorRunner.java

```diff
@@ -81,24 +77,13 @@ public void init() {
   }
 
   public void destroy() {
+    LOGGER.info("shutting down replicatorRunner");
     scheduledExecutor.shutdownNow();
   }
 
   @VisibleForTesting
   void replicateAsSystemUser() {
```

Comment:
We should probably just remove this method and directly call scheduleReplication()